.github/workflows/spark_test.yaml

```diff
@@ -41,6 +41,10 @@ jobs:
           sudo apt-get update
           sudo apt-get install -y make build-essential libssl-dev zlib1g-dev libbz2-dev libreadline-dev libsqlite3-dev wget curl llvm libncurses5-dev libncursesw5-dev xz-utils tk-dev libffi-dev liblzma-dev python-openssl git
           sudo apt install libedit-dev
+          curl -LO https://github.com/bufbuild/buf/releases/download/v1.28.1/buf-Linux-x86_64.tar.gz
```

Comment:
Do we need to add this in the requirements?

Reply:
I think not because the `requirements.txt` is only for Python dependencies, used for the Python `venv`